### DISCUSSION TITLE
Option to prevent file reopening

### DISCUSSION THREAD

**Discussion by minsk21** [sumatrapdfreader/sumatrapdf#3527] [Ideas]
If the file is already open, then a window opens with the already open one, not a new one.

**Comment by GitHubRulesOK** [comment]
depends what settings are involved the default may be reuse false so try setting it to  true

![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/63143b9c-3b45-45d8-aefe-21a0557e5694)

**Reply by minsk21** [reply]
It's works. Thanks
